### PR TITLE
add missing space - Part 2

### PR DESCRIPTION
**Pull request by mairaw**

### CODE REVIEW

File: samples/snippets/visualbasic/VS_Snippets_Remoting/BindingCollectionSample2/VB/bindingcollectionsample2.vb [hunk 1]
```diff
@@ -6,14 +6,14 @@
 
 ' Grouping Clause : Snippet5 and Snippet8 go together.
 
-' The following example demonstrates the constructor 'Binding()' and properties 'Extensions','Name','Operations',
-'  'ServiceDescription' and 'Type' property of 'Binding' class AND method 'Insert' of 'BindingCollection' class.
+' The following example demonstrates the constructor 'Binding()' and properties 'Extensions', 'Name', 'Operations',
+'  'ServiceDescription', and 'Type' property of 'Binding' class AND method 'Insert' of the 'BindingCollection' class.
```
Comment (bartonjs) [inline samples/snippets/visualbasic/VS_Snippets_Remoting/BindingCollectionSample2/VB/bindingcollectionsample2.vb:10]:
```suggestion
' The following example demonstrates the constructor 'Binding()' and the 'Extensions', 'Name', 'Operations',
'  'ServiceDescription', and 'Type' properties of 'Binding' class as well as the 'Insert' method of the 'BindingCollection' class.

```

File: samples/snippets/visualbasic/VS_Snippets_Remoting/NetworkCredential_UserName_Password_Domain/VB/networkcredential_username_password_domain.vb [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
  'This program demontrates the 'UserName','Domain' and 'Password' properties of 'NetworkCredential' class.
 '  It takes an URL, username, password and domainname from console. An empty 'NetworkCredential' object 
-'  is created.The 'UserName' ,'Password' and 'Domain' porperties of 'NetworkCredential' class are initialised 
+'  is created. The 'UserName' ,'Password' and 'Domain' porperties of 'NetworkCredential' class are initialised
```
Comment (bartonjs) [inline samples/snippets/visualbasic/VS_Snippets_Remoting/NetworkCredential_UserName_Password_Domain/VB/networkcredential_username_password_domain.vb:5]:
```suggestion
'  is created. The 'UserName' ,'Password' and 'Domain' properties of 'NetworkCredential' class are initialised 

```

File: xml/System.Linq/ParallelEnumerable.xml [hunk 3]
```diff
@@ -1220,7 +1220,7 @@
         <returns>The average of the sequence of values.</returns>
         <remarks>To be added.</remarks>
         <exception cref="T:System.OperationCanceledException">The query was canceled with the token passed in through <paramref name="WithCancellation" />.</exception>
-        <exception cref="T:System.ArgumentNullException">A sequence of values that are used to calculate an average.The average of the sequence of values.<paramref name="source" /> is a null reference (Nothing in Visual Basic).</exception>
+        <exception cref="T:System.ArgumentNullException">A sequence of values that are used to calculate an average. The average of the sequence of values.<paramref name="source" /> is a null reference (Nothing in Visual Basic).</exception>
```
Comment (bartonjs) [inline xml/System.Linq/ParallelEnumerable.xml:1223]:
```suggestion
        <exception cref="T:System.ArgumentNullException"><paramref name="source" /> is a null reference (Nothing in Visual Basic).</exception>
```

or

```suggestion
        <exception cref="T:System.ArgumentNullException"><paramref name="source" /> is a <see langword="null"/>.</exception>
```

File: samples/snippets/visualbasic/VS_Snippets_Remoting/NetworkCredential_UserName_Password_Domain/VB/networkcredential_username_password_domain.vb (hunk 2 above)
Comment (mairaw) [reply samples/snippets/visualbasic/VS_Snippets_Remoting/NetworkCredential_UserName_Password_Domain/VB/networkcredential_username_password_domain.vb:5]: Yes, we do.

File: xml/System.Linq/ParallelEnumerable.xml (hunk 3 above)
Comment (mairaw) [reply xml/System.Linq/ParallelEnumerable.xml:1223]: Good catch. I didn't know you could have more than one suggestion like that. Cool!

**Pull request merged**